deploy/csi-hostpath-plugin.yaml

```diff
@@ -60,13 +60,13 @@ spec:
               name: csi-data-dir
 
         - name: hostpath
-          image: docker.io/gmontero/origin-projected-resource-csi-driver:latest
+          image: docker.io/gmontero/origin-csi-driver-projected-resource:latest
```

Comment:
Don't we want to reference the quay image here?

Reply:
Yep, i missed that one, thanks.